sections/loans/components/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,293 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ethers } from 'ethers';
+import { useRecoilValue } from 'recoil';
+
+import { walletAddressState } from 'store/wallet';
+import { ExternalLink } from 'styles/common';
+import styled from 'styled-components';
+import Connector from 'containers/Connector';
+import Button from 'components/Button';
+import Currency from 'components/Currency';
+import { Synths } from 'constants/currency';
+import synthetix from 'lib/synthetix';
+import { tx } from 'utils/transactions';
+import { formatNumber, toBigNumber, formatUnits } from 'utils/formatters/number';
+import { useLoans } from 'sections/loans/contexts/loans';
+import InfoSVG from 'sections/loans/components/ActionBox/components/InfoSVG';
+
+const InfoBox: React.FC = () => {
+	const { t } = useTranslation();
+	const address = useRecoilValue(walletAddressState);
+	const { provider } = Connector.useContainer();
+	const { pendingWithdrawals, reloadPendingWithdrawals, ethLoanContract } = useLoans();
+	const [isClaimingPendingWithdrawals, setIsClaimingPendingWithdrawals] = React.useState(false);
+
+	const [borrows, setBorrows] = React.useState<Array<any>>([]);
+	const borrowsOpenInterest = React.useMemo(
+		() => borrows.reduce((sum, stat) => sum.plus(stat.openInterest), toBigNumber(0)),
+		[borrows]
+	);
+
+	const claimPendingWithdrawals = async () => {
+		if (!ethLoanContract) return;
+		try {
+			setIsClaimingPendingWithdrawals(true);
+			const pw = await ethLoanContract.pendingWithdrawals(address);
+			await tx(() => [ethLoanContract, 'claim', [pw]]);
+			await reloadPendingWithdrawals();
+		} catch {
+		} finally {
+			setIsClaimingPendingWithdrawals(false);
+		}
+	};
+
+	React.useEffect(() => {
+		if (!provider) {
+			return;
+		}
+
+		const {
+			contracts: {
+				ExchangeRates: exchangeRatesContract,
+				CollateralManager: collateralManagerContract,
+			},
+		} = synthetix.js!;
+
+		let isMounted = true;
+		const unsubs: Array<Function> = [() => (isMounted = false)];
+
+		const getBorrowStats = async (currency: string) => {
+			const [openInterest, [assetUSDPrice]] = await Promise.all([
+				collateralManagerContract.long(ethers.utils.formatBytes32String(currency)),
+				exchangeRatesContract.rateAndInvalid(ethers.utils.formatBytes32String(currency)),
+			]);
+			const openInterestUSD = toBigNumber(openInterest.toString())
+				.dividedBy(1e18)
+				.multipliedBy(toBigNumber(assetUSDPrice.toString()).dividedBy(1e18));
+			return {
+				currency,
+				openInterest: openInterestUSD,
+			};
+		};
+
+		const loadBorrowsStats = () =>
+			Promise.all([Synths.sBTC, Synths.sETH, Synths.sUSD].map(getBorrowStats));
+
+		const load = async () => {
+			try {
+				const borrows = await loadBorrowsStats();
+				if (isMounted) {
+					setBorrows(borrows);
+				}
+			} catch (e) {
+				console.error(e);
+			}
+		};
+
+		const subscribe = () => {
+			const newBlockEvent = 'block';
+			provider!.on(newBlockEvent, load);
```

Comment:
calling `load()` on every block might be a bit aggressive. it's every 10-15secs. I think we should really avoid this if possible. Do you think it's possible?